superdesk/metadata/item.py

```diff
@@ -285,6 +285,19 @@
         'type': 'string',
         'nullable': True,
     },
+    'authors': {
+        'type': 'list',
+        'nullable': True,
+        'mapping': {
+            'type': 'object',
+            'properties': {
+                'parent': not_analyzed,
```

Comment:
who is authors parent?

Reply:
it's the user (user id). Parent was being used in client for subjects to have the tree, so I took the same mechanism (parent are users, children are roles).